src/ChatTextToSpeech.csproj

```diff
@@ -1,13 +1,13 @@
-<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFramework>netcoreapp2.1</TargetFramework>
     <AzureFunctionsVersion>v2</AzureFunctionsVersion>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="Azure.AI.TextAnalytics" Version="1.0.0-preview.4" />
-    <PackageReference Include="Azure.Storage.Blobs" Version="12.4.2" />
-    <PackageReference Include="Microsoft.Azure.WebJobs.Extensions.EventHubs" Version="3.0.0" />
-    <PackageReference Include="Microsoft.NET.Sdk.Functions" Version="1.0.24" />
+    <PackageReference Include="Azure.AI.TextAnalytics" Version="5.0.0" />
+    <PackageReference Include="Azure.Storage.Blobs" Version="12.5.0-preview.6" />
```

Comment:
although this is latest, I wonder if it is better to have the sample with a GA package.... like you had this one before

Reply:
Let's only use preview if there are features in the preview that are needed for the sample to run.

Reply:
@tzhanl - Please address.

Reply:
copy that. Will follow this rule.

Reply:
Resolved, updated to package 12.4.4